calamari-common/calamari_common/remote/mon_remote.py

```diff
@@ -1057,3 +1065,30 @@ def tail(subpath, n_lines):
     p = subprocess.Popen(["tail", "-n", str(n_lines), path], stdout=subprocess.PIPE)
     stdout, stderr = p.communicate()
     return stdout
+
+
+def cluster_stats(cluster_name):
+    import rados
+
+    cluster = rados.Rados(name=RADOS_NAME, clustername=cluster_name, conffile='')
```

Comment:
Use a context manager like https://github.com/ceph/calamari/blob/89147575998cfcb7318bfb74621c9cca784fb8ec/calamari-common/calamari_common/remote/mon_remote.py#L77 to prevent memory leaks. mon_remote.py is long-running code